### DISCUSSION TITLE
configMigration and automerge

### DISCUSSION THREAD

**Discussion by hoepfnerj** [renovatebot/renovate#23302] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.x

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I am trying to use the configuration options `configMigration` and `automerge` together. So Renovate should migrate the `renovate.json5` configuration in the target repository and merge it without waiting for PR approval.

### What you tried so far.

I have tried the following configuration, but it does not seem to work:

In the `config.js`:
```
...
configMigration: true,
...
```
and in the `package.json5` file in the target directory:
```
packageRules": [
    {
      "matchFileNames": ["renovate.json5"],
      "automerge": true,
      "automergeType": "branch",
      "ignoreTests": true
    }
]
```
Is this currently possible? I see that the PR that is created has no "Configuration" section in the Overview.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Do you mean you want Renovate to automatically merge the PR which migrates its own config?

**Reply by hoepfnerj** [reply]
Yes, exactly. That would be helpful, since we have a lot of repositories.

**Reply by rarkins** [reply, marked as answer]
I added feature request #23317
